Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.yaml

```
@@ -4,37 +4,36 @@ metadata:
   name: spark-fts-{{ts_nodash|replace("T", "")}}
 spec:
   sparkConf:
-    spark.mapr.extraconf.secret: {{dag_run.conf.get("s3_secret_name", "spark-s3-creds")}}
     spark.hadoop.fs.s3a.path.style.access: "true"
     spark.hadoop.fs.s3a.impl: "org.apache.hadoop.fs.s3a.S3AFileSystem"
     spark.hadoop.fs.s3a.connection.ssl.enabled: "true"
+    spark.hadoop.fs.s3a.access.key: <access-token>
```

For SparkApplication yaml files in Airflow DAGs we couldn't leave some random placeholders, because Airflow doesn't know how to parse `<access-token>`. I guess you suppose that user will manually edit and paste here his JWT, but in Airflow user couldn't do it, because DAGs are pulled directly from git repo.

I propose to add a new param in python DAG file:
```python3
        'jwt_token': Param("", type="string"),
```

And use here a template for this param:
```suggestion
    spark.hadoop.fs.s3a.access.key: {{dag_run.conf.get("jwt_token", "")}}
```